cmd/deploy/deploy.go

```diff
@@ -185,9 +182,8 @@ func Deploy(ctx context.Context) *cobra.Command {
 	}
 
 	cmd.Flags().StringVar(&options.Name, "name", "", "application name")
-	cmd.Flags().StringVarP(&options.ManifestPath, "file", "f", "", "path to the manifest file")
-	cmd.Flags().StringVar(&options.Namespace, "namespace", "", "application name")
-	cmd.Flags().StringVar(&options.K8sContext, "context", "", "k8s context")
+	cmd.Flags().StringVarP(&options.ManifestPath, "file", "f", "", "path to the okteto manifest file")
+	cmd.Flags().StringVarP(&options.Namespace, "namespace", "n", "", "overwrites the namespace where the application is deployed")
```

Comment:
What is being overridden here and is it obvious to a cli user what they are overriding?

Reply:
Yes, we have this flag in all our CLI commands. It allows to deploy the app in a different namespace than the one of the current okteto context. What isn't clear? How would you rephrase it?

Reply:
It's not obvious to me where the original namespace exists. I don't know if it's in the manifest file or part of my context. How would a user know what there current namespace is and if it's something they want to override. I would probably state where the namespace being overwritten is?